components/layout/fragment.rs

```diff
@@ -186,6 +186,8 @@ pub enum SpecificFragmentInfo {
     Multicol,
     MulticolColumn,
     UnscannedText(Box<UnscannedTextFragmentInfo>),
+
```

Comment:
It's not necessarily text, though. Non-text fragments get truncated, but in an all-or-nothing fashion.